collections/_priorities/assetio-04-fix-scene-inheritance.md

```diff
@@ -0,0 +1,14 @@
+---
+last_updated: "2024-11-30"
+category: assetio
+rank: 4
+created: "2024-11-30"
+anchor: "fix-scene-inheritance"
+title: "Fix scene inheritance"
+description: |
+  Scene inheritance is the concept of creating a scene based on an existing one. It is a feature that should work well in theory, but can be quite finicky in reality.
+details:
+  - type: note
+    content: |
+      One idea is to give nodes unique IDs.
```

Comment:
See the discussion here: https://github.com/KhronosGroup/glTF/issues/2337

TODO:
* Achieve agreement between Khronos, Godot, Blender, and any other organizations for if this feature is desired, when it should and should not be used, what the design goals are, what it should look like, etc.
* Write a draft specification that details the above information.
* Implement a prototype of the feature in Godot.
* Implement a prototype of the feature in Blender.